onnx/test/optimizer_test.py

```diff
@@ -707,6 +707,97 @@ def test_lift_lex_if(self):  # type: () -> None
         assert optimized_model.graph.node[2].attribute[2].strings[0] == b"X"
         assert optimized_model.graph.node[2].attribute[2].strings[1] == b"Y"
 
+    def test_fuse_bn_into_conv_simple_float(self):  # type: () -> None
+        conv = helper.make_node("Conv", ["X", "W", "B"], ["Y"])
+        bn = helper.make_node("BatchNormalization", ["Y", "scale", "b", "mean", "var"], ["Z"])
+        graph = helper.make_graph(
+            [conv, bn],
+            "test",
+            [helper.make_tensor_value_info("X", TensorProto.FLOAT, (5, 2, 28, 28)),
+             helper.make_tensor_value_info("W", TensorProto.FLOAT, (3, 2, 5, 5)),
+             helper.make_tensor_value_info("B", TensorProto.FLOAT, (3,)),
+             helper.make_tensor_value_info("scale", TensorProto.FLOAT, (3,)),
+             helper.make_tensor_value_info("b", TensorProto.FLOAT, (3,)),
+             helper.make_tensor_value_info("mean", TensorProto.FLOAT, (3,)),
+             helper.make_tensor_value_info("var", TensorProto.FLOAT, (3,))],
+            [helper.make_tensor_value_info("Z", TensorProto.FLOAT, (3,))],
+            [helper.make_tensor("W", TensorProto.FLOAT,
+                                dims=(3, 2, 5, 5),
+                                vals=np.random.randn(3, 2, 5, 5).astype(np.float32).tobytes(),
+                                raw=True),
+            helper.make_tensor("B", TensorProto.FLOAT,
+                                dims=(3,),
+                                vals=np.random.randn(3,).astype(np.float32).tobytes(),
+                                raw=True),
+            helper.make_tensor("scale", TensorProto.FLOAT,
+                                dims=(3,),
+                                vals=np.random.randn(3,).astype(np.float32).tobytes(),
+                                raw=True),
+            helper.make_tensor("b", TensorProto.FLOAT,
+                                dims=(3,),
+                                vals=np.random.randn(3,).astype(np.float32).tobytes(),
+                                raw=True),
+            helper.make_tensor("mean", TensorProto.FLOAT,
+                                dims=(3,),
+                                vals=np.random.randn(3,).astype(np.float32).tobytes(),
+                                raw=True),
+            helper.make_tensor("var", TensorProto.FLOAT,
+                                dims=(3,),
+                                vals=np.random.randn(3,).astype(np.float32).tobytes(),
+                                raw=True)],
+        )
+        optimized_model = self._optimized(graph, ["fuse_bn_into_conv"])
+
+        assert len(list(optimized_model.graph.node)) == 1
```

Comment:
use `self.assertEqual`, which will print the mismatched values in case the test failed.

Reply:
added loop and assertEqual